ncclient/xml_.py

```diff
@@ -223,11 +223,17 @@ def remove_namespaces(self, rpc_reply):
                                        parser=self.__parser)
         return self.__root
 
+def parent_ns(node):
+    index1 = node.tag.find('{')
+    index2 = node.tag.find('}')
+    if index1 != -1 and index2 != -1 and index1 < index2:
+        return node.tag[index1+1: index2]
+    return None
```

Comment:
Why not use built-in capabilities of lxml for this? It already parses namespaces out. For example:

```python
def parent_ns(node):
    if node.prefix:
        return node.nsmap[node.prefix]
    return None
```

Suggest you update `parent_ns` to this.

Reply:
Thank you for your review, I've updated it :)